doc/cephfs/disaster-recovery-experts.rst

```diff
@@ -66,40 +66,44 @@ store state.
 Journal truncation
 ------------------
 
-If the journal is corrupt or MDSs cannot replay it for any reason, you can
-truncate it like so:
+Use a command of the following form to truncate any journal that is corrupt or
+that an MDS cannot replay:
 
-::
+.. prompt:: bash #
 
-    cephfs-journal-tool [--rank=<fs_name>:{mds-rank|all}] journal reset --yes-i-really-really-mean-it
+   cephfs-journal-tool [--rank=<fs_name>:{mds-rank|all}] journal reset --yes-i-really-really-mean-it
 
-Specify the filesystem and the MDS rank using the ``--rank`` option when the file system has/had
-multiple active MDS.
+Specify the filesystem and the MDS rank using the ``--rank`` option when the
+file system has or had multiple active MDS.
 
 .. warning::
 
-    Resetting the journal *will* lose metadata unless you have extracted
-    it by other means such as ``recover_dentries``.  It is likely to leave
-    some orphaned objects in the data pool.  It may result in re-allocation
-    of already-written inodes, such that permissions rules could be violated.
+    Resetting the journal *will* lose metadata unless you have extracted it by
+    other means such as ``recover_dentries``. Resetting the journal is likely
+    to leave some orphaned objects in the data pool. Resetting the journal may
```

Comment:
s/some//

Reply:
I agree. Now that I'm looking at this with fresh eyes, I think that I'm going to change the sentence to read "Resetting the journal *will* cause metadata to be lost unless you have extracted it.."